build.gradle

```diff
@@ -14,7 +14,7 @@ ext {
     androidxWorkVersion = "2.4.0"
 
     daggerVersion = '2.29.1'
-    fluxCVersion = '1.32.0'
+    fluxCVersion = '1.33.0'
```

Comment:
Note: from what I understand from the list of PRs which landed in FluxC between `1.32.0` and the `1.33.0` I created during that release, all the PRs there were about WooCommerce and not WordPress.

In fact, since `develop` of WPAndroid was still pointing to `fluxCVersion = '1.32.0'` before code-freeze, that suggests that the app was already happy with FluxC `1.32.0` (no PR in WPAndroid had to point to an intermediate `develop-<sha1>` commit of the library that would be past `1.32.0`), so that bump was probably not strictly necessary for WPAndroid's code freeze. 🤷 

But eh. I say better always point to the latest (in case some bugs were fixed in between by the Woo team that WP could benefit from)

Reply:
> But eh. I say better always point to the latest (in case some bugs were fixed in between by the Woo team that WP could benefit from)

What about new bugs that might have been introduced between 1.32.0 and 1.33.0... :trollface: 

Jokes aside, I agree 👍 Also, not bumping here would have brought up the question "why is this a two version bump?" at the next release time, which might have been confusing.